lib/helper/WebDriverIO.js

```diff
@@ -924,7 +1002,71 @@ class WebDriverIO extends Helper {
       }).then((selected) => {
         if (!Array.isArray(selected)) selected = [selected];
         selected = selected.filter((val) => val === true);
-        return assert.equal(selected.length, num, `expected nummber of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+        return assert.equal(selected.length, num,
+          `expected number of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+      });
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given CSS property.
+   */
+  seeCssPropertyOnElement(locator, cssProperty, value) {
```

Comment:
I think we should not allow to test several properties in one step. Current implementation takes all elements with given locator and request cssProperty. If all elements will not have expected cssproperty value, then we will throw the error. If we will give opportunity to validate several props in one steps there may appear false positive errors